about/glossary.qmd

```diff
@@ -0,0 +1,65 @@
+---
+title: "Glossary of key concepts"
+---
+
+## Session 1: Statistical concepts
+
+**Probability density function**
+
+**Probability mass function**
+
+**Sampling the posterior**
+
+
+
+### Session 2
+
+**Primary and secondary events**: when one epidemiological event (the primary event) directly causes another epidemiological event (the secondary event).
+For example, infection and symptom onset; infection in one person causing infection in another.
+A focus of this course is on understanding the time between these pairs of events.
+Key pairs of events include: 
+- The incubation period: infection to symptom onset 
+- Symptom onset to hospitalisation 
+- The generation time: infection in one person causing infection in another
+
+### Session 3
+
+**Censoring**: knowing that an event occurred at some time, but not specifically when.
+In this course, we mostly consider double (or interval) censored events: we know the event occurred at some time within a time interval, such as at any time on a certain date.
+Events might also be right censored (we only know the earliest time it could have happened) or left censored (we only know the latest time it could have occurred).
```

Comment:
I think these are the wrong way around? I think if something is right censored we know events up until a latest time and then beyond that we know they happened but not the value.